modules/doc/training/content/sqa/training/ccb/pln-4005.md

```diff
@@ -0,0 +1,187 @@
+  resources exist (time, talent, funding, etc.)
+- +Project Lead+: Manages day-to-day development of the code to ensure that work is being performed
+  as defined by the Line Manager, and performs software releases
+- +Software Quality Assurance Lead+: Manages [!ac](SQA) processes and ensures that PLN-4005 is
+  being implemented
+- +Change Control Board (CCB)+: Perform reviews of Change Requests (CRs) to ensure that code
+  changes are necessary and that documentation and testing is performed as defined in PLN-4005
+
+!---
+
+## PLN-4005: Documentation
+
+Three classes of documentation are defined for MOOSE and MOOSE-based applications:
+
+- static
+- repository
+- development
+
+!---
+
+### PLN-4005: Static Documentation
+
+- +Software Quality Assurance Plan (SQAP; PLN-4005)+: Defines [!ac](SQA)
+  processes implemented including Configuration Management Plan (CMP) and Verification
+  and Validation Plan (VVP)\*.
+- +Safety-software Determination (SSD)+: [!ac](INL) form to declare if the application is safety
+  software
+- +Quality-level Determination (QLD)+: [!ac](INL) form to declare the safety software level
+- +[!ac](EA) Entry+: [!ac](INL) from that lists [!ac](SQA) documentation for application
+
+!alert note
+The SDD, QLD, and EA Entry are the only [!ac](INL) level documents that are necessary for satisfying the
+NQA-1 standard and are that is created when the [!ac](SQA) process begins.
+
+\**VVP is referring verification and validation of the NQA-1 process, not the science.*
+
+!---
+
+### PLN-4005: Repository Documentation
+
+All documentation stored in the repository, the following [!ac](SQA) specific documents must be
+included to conform to PLN-4005.
+
+- Software requirements specifications (SRS)
+- Software design descriptions (SDD)
+- Requirements Traceability Matrix (RTM)
+- Software test plan (STP)
+- Failure-analysis report (FAR)
+- Communication and contact information (CCI)
+- Software coding standards (SCS)
+
+!---
+
+### PLN-4005: Development Documentation
+
+The following records shall be maintained to document the change control process leading
+to a release:
+
+- Change Requests (Pull/Merge Requests)
+- Issues (GitHub/GitLab)
+- Test results (CIVET)
+- Release logs (create as part of release)
+
+!---
+
+## PLN-4005: Reviews
+
+- +Static Documentation Review+: Changes to the static documentation follow the review procedure
+  native to the INL system (SQA Lead should handle these)
+
+- +Design Verification Review+: "A review shall be conducted by an independent reviewer. The reviewer
+  may rely on assistance from any individual, but the final approval or disapproval is the sole
+  responsibility of the independent reviewer. All comments are retained in the GitHub/GitLab system.
+
+- +Release Review+: A release requires an additional release review...to ensure compliance with the
+  approved software requirements. Test results from CIVET are considered part of the review.
```

Comment:
The ellipsis looks weird here - should this be a quote?

Reply:
Yes, but it doesn't matter.